Gemfile

```diff
@@ -14,7 +14,8 @@ branch = File.read(File.expand_path("../maintenance-branch", __FILE__)).chomp
 end
 
 ### dep for ci/coverage
-gem 'simplecov', '~> 0.8'
+gem 'simplecov', '~> 0.9'
+gem 'simplecov-html', :github => 'colszowka/simplecov-html'
```

Comment:
One of your comments mentioned this addresses a failure but I don't understand...we didn't need this gem before so why do we need it now? What failure does it address?